### ISSUE TITLE
MO ongoing support pt. 3 (blocked)

### ISSUE THREAD

**Issue by sychang** [Recidiviz/pulse-dashboard#405]
An extension of MO ongoing support pt. 1: [#398](https://app.zenhub.com/workspaces/recidiviz-corelantern-5ec36552cdfe7bc58f04bdbe/issues/recidiviz/pulse-dashboard/398) and pt. 2 [#406](https://app.zenhub.com/workspaces/recidiviz-corelantern-5ec36552cdfe7bc58f04bdbe/issues/recidiviz/pulse-dashboard/406) consisting of items that are blocked on FE complexity resolution or dependent on scoping.

[WIP MO ongoing product support strategy doc
](https://docs.google.com/document/d/1HLRRxggyMgl_repc7DV2OU3vLIYM_FkyCnAt4KwPiuw/edit#)

This is the prioritized list of requests from MO excluding things that are in pt 1, ranked from top to bottom in terms of priority to them:

5 (five). [Blocked on MO and Serena] For district administrators and unit supervisors, show only the state-wide view and their district. Lock them out of other districts. TODO: Long-term, needs scoping
9 (nine). [Blocked on MO] Change revocation cluster lookback period to 9 months (or generally make it shorter). TODO: @sychang file an issue
10 (ten). [Blocked] Include all 20+ case types, including things like electronic monitoring, supervision level breakouts, initial assessment phase, deferred prosecution, supervision level 1, level 2, level 3, etc. #397
11 (eleven). [Blocked] Enable custom date range selection. #335
12 (twelve). [Blocked] Revocation cluster lookback period: ability to filter by shorter lookback period by 1 mo, 3 mos, 6 mos, 9 mos, 12 mos, and custom date range. TODO: @sychang file an issue

**Comment by sychang** [comment]
Closing because replaced this master epic with epics for the individual items in here.
